### ISSUE TITLE
Update UI Design for Debtor Group Management and Creditor Group Management Modules

### ISSUE THREAD

**Issue by lomamech** [Third-Culture-Software/bhima#7680]
Description:

**The Debtor Group Management** and **Creditor Group Management** are currently using the outdated BHIMA 1.x design. To improve the user experience and maintain consistency with the rest of the system, these modules should be updated to use a UiGrid component. The updated UI should include:

Filter options for easier data management.
A Search button to quickly locate specific entries.
Column management capabilities to customize the display of data.
This change will enhance usability and align these modules with the modern BHIMA design standards.

**Comment by jniles** [comment]
I agree with this proposal.   They aren't old 1.x designs (we introduced them in 2.x), but they didn't live up to the promise of making the interfaces more user friendly.  The old grid style is much more functional.
